app/test_split_integer.py

```diff
@@ -2,20 +2,31 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value = 10
+    num_parts = 5
+    res = split_integer(value, num_parts)
+    assert sum(res) == value
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    expected = split_integer(10, 5)
```

Comment:
According to the checklist, avoid creating unnecessary variables for simple tests. You can directly assert the function call result with the expected list: `assert split_integer(10, 5) == [2, 2, 2, 2, 2]`.